src/telcameras_v2/ingress_parser.py

```diff
@@ -2,20 +2,31 @@
 import logging
 
 from django.conf import settings
+from ingress.consumer.base import BaseConsumer
 
-from ingress.parser import IngressParser
 from telcameras_v2.serializers import ObservationSerializer
 from telcameras_v2.tools import (SensorError, data_to_observation,
                                  get_sensor_for_data)
 
 logger = logging.getLogger(__name__)
 
 
-class TelcameraParser(IngressParser):
-    endpoint_url_key = 'telcameras_v2'
+class TelcameraParser(BaseConsumer):
+    collection_name = 'telcameras_v2'
 
-    def parse_single_message(self, ingress_raw_data):
-        data = json.loads(ingress_raw_data)['data']
+    """
+    Whether or not to immediately remove messages once consumption succeeds.
+    If set to False, message.consume_succeeded_at will be set.
+    """
+    remove_message_on_consumed = False
+
+    """
+    Whether or not to set Message.consume_started_at immediately once consumption starts
+    """
+    set_consume_started_at = True
```

Comment:
Wat zou de use case zijn om dit op False te zetten?

Reply:
Wanneer `remove_message_on_consumed == True`. In dat geval weet je al dat je hem gaat verwijderen, en is de extra roundtrip naar de db om de started_at op te slaan niet nodig.